things/api/things/http/endpoint_test.go

```diff
@@ -2407,6 +2407,198 @@ func TestCreateConnections(t *testing.T) {
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	thIDs := []string{}
+	for _, th := range ths {
+		thIDs = append(thIDs, th.ID)
+	}
+
+	chs, err := svc.CreateChannels(context.Background(), token, channel)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	chIDs1 := []string{}
+	for _, ch := range chs {
+		chIDs1 = append(chIDs1, ch.ID)
+	}
+
+	chs, err = svc.CreateChannels(context.Background(), otherToken, channel)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	chIDs2 := []string{}
+	for _, ch := range chs {
+		chIDs2 = append(chIDs2, ch.ID)
+	}
+
+	err = svc.Connect(context.Background(), token, chIDs1, thIDs)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+
+	cases := []struct {
+		desc        string
+		channelIDs  []string
+		thingIDs    []string
+		auth        string
+		contentType string
+		body        string
+		status      int
+	}{
+		{
+			desc:        "disconnect from existing things to existing channels",
+			channelIDs:  chIDs1,
+			thingIDs:    thIDs,
+			auth:        token,
+			contentType: contentType,
+			status:      http.StatusOK,
+		},
+		{
+			desc:        "disconnect from existing things to non-existent channels",
+			channelIDs:  []string{strconv.FormatUint(wrongID, 10)},
+			thingIDs:    thIDs,
+			auth:        token,
+			contentType: contentType,
+			status:      http.StatusNotFound,
+		},
+		{
+			desc:        "disconnect from non-existing things to existing channels",
+			channelIDs:  chIDs1,
+			thingIDs:    []string{strconv.FormatUint(wrongID, 10)},
+			auth:        token,
+			contentType: contentType,
+			status:      http.StatusNotFound,
+		},
+		{
+			desc:        "disconnect from existing things to channel with invalid id",
+			channelIDs:  []string{"invalid"},
+			thingIDs:    thIDs,
+			auth:        token,
+			contentType: contentType,
+			status:      http.StatusNotFound,
+		},
+		{
+			desc:        "disconnect from things with invalid id to existing channels",
+			channelIDs:  chIDs1,
+			thingIDs:    []string{"invalid"},
+			auth:        token,
+			contentType: contentType,
+			status:      http.StatusNotFound,
+		},
+		{
+			desc:        "disconnect from existing things to empty channel ids",
+			channelIDs:  []string{""},
+			thingIDs:    thIDs,
+			auth:        token,
+			contentType: contentType,
+			status:      http.StatusBadRequest,
+		},
+		{
+			desc:        "disconnect from empty things id to existing channels",
+			channelIDs:  chIDs1,
+			thingIDs:    []string{""},
+			auth:        token,
+			contentType: contentType,
+			status:      http.StatusBadRequest,
+		},
+		{
+			desc:        "disconnect from existing things to existing channels with invalid token",
+			channelIDs:  chIDs1,
+			thingIDs:    thIDs,
+			auth:        wrongValue,
+			contentType: contentType,
+			status:      http.StatusUnauthorized,
+		},
+		{
+			desc:        "disconnect from existing things to existing channels with empty token",
```

Comment:
Disconnect existing things from existing channels...
Please review descriptions for all the other test cases, as well.